README.md

```diff
@@ -62,23 +62,54 @@ An example usage is by getting the list of PMID's from ```get_list_pmid``` and t
 
 ### Command line
 
-Can be run on Command line with the command
+Can be run on Command line with the command, look below for required arguments
 
 ```bash
-pubmed-dl
+main.py --help

```

Comment:
Not sure why we changed this back to `main.py`? `pubmed-dl` is better because once a user installs our package they can call it from anywhere. Calling just `main.py` assumes they have cloned this locally and are in the correct folder.

Reply:
I'm not able to run it as pubmed-dl, is that because I havent downloaded it as a package?

Reply:
how would the commands run then? `pubmed-dl --help` ??

Reply:
Yes, although something has broke the `pubmed-dl` command in this PR. I will have to debug later.

Reply:
I never used the `pubmed-dl` command before so I'm not sure where to check for that issue. I would appreciate your help on this matter.

Reply:
Just merge and open another issue to track this, I'll have to get to it later